### DISCUSSION TITLE
How to View Particle Tracks Only in Certain Regions

### DISCUSSION THREAD

**Discussion by barghashr** [nrc-cnrc/EGSnrc#1070] [Questions]
Hi folks,

How can I view particle tracks in certain (preferred) regions only? for example, viewing only particles tracks from within a certain region or geometry and eliminating the ones in outer regions or geometries. 

Thank you!

**Comment by rtownson** [comment]
There's no easy way to do this, currently. In theory, you could restrict the x,y,z positions within which to display tracks either in egs_view (in HEN_HOUSE/egs++/view/egs_track_view.cpp), or restrict what is written to the tracks file by editing egs_track_scoring. The latter is probably the better choice, you could check the region number fairly easily from the ausgab object code. If you're interested to do some coding and add this as a feature, let me know and I can provide some more tips.
